src/index.html

```diff
@@ -7,7 +7,78 @@
     <title>Product cards</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Stars</h1>
+  <body class="body">
+    <div>
+      <div class="stars stars--0">
+        <div class="stars__star"></div>
+
+        <div class="stars__star"></div>
+
+        <div class="stars__star"></div>
+
+        <div class="stars__star"></div>
+
+        <div class="stars__star"></div>
```

Comment:
it's ok according to the checklist, but further you can  do not leave empty lines if the tag contains only 1 line